### ISSUE TITLE
Add Emote and other Common Animations (walking, sitting, etc)

### ISSUE THREAD

**Issue by lowlines** [Bungie-net/api#243]
In Destiny 1, a handful of Eververse emotes were included with their animations for previewing on the website. My understanding was this was all manually set up and then abandoned due to time investment involved with maintaining it.

If possible it would be good to get these back again and in a more standardised state for the Companion app and when third party tools like my TGX Loader support them.

**Comment by vthornheart-bng** [comment]
Yes, I would very much like to automate this process so we start getting this data back more consistently.  I don't know when we'll be able to get to it, but I'm glad it's on the radar.

**Comment by ckhicks** [comment]
@lowlines Do you mean the naming conventions or files being made available at all? I dug through the wayback machine a couple months ago looking for Eververse previews before the big push at the end of D1 and found some fragments that helped get (some) of the animation files.

**Comment by vthornheart-bng** [comment]
Yeah, ideally both: making the files themselves available, and stitching it into the existing definitions so that they're easy to find.

**Comment by vthornheart-bng** [comment]
There's really no reason for us *not* to do so, and if people can do cool stuff with it... I mean, that's great!  We just need a better pipeline on our backend, right now it's a bit of a manual process to extract and return that data, we need to automate it before we reintroduce it.

**Comment by lowlines** [comment]
@ckhicks There was actually a property on D1 InventoryItemDefinitions for the animation file. I got a list of them.
https://github.com/DestinyDevs/BungieNetPlatform/blob/master/three-tgx-loader/gistfile1.txt

**Comment by PepijnAverink** [comment]
@vthornheart-bng Any updates on this so far? I would love this feature in the D2 api.

**Comment by lowlines** [comment]
@jshaffstall-bng do you think you could revisit this along with my suggestion about basic animations/skeleton for weapons and things like ghost shells?

`https://bungie/net/common/destiny_content/animations/destiny_player_skeleton.js` and `https://bungie/net/common/destiny_content/animations/destiny_player_animation.js` should get added to Destiny 2 content with some way to reference it from the manifest response, ie `mobileGearCDN.animation+mobileGearPaths.player_skeleton`

```
{
  ...
  mobileGearCDN: {
    ...
    animation: "/common/destiny_content/animations"
  },
  mobileGearPaths: {
    player_skeleton: "destiny_player_skeleton.js",
    player_animation: "destiny_player_animation.js",
    ghost_skeleton: "destiny_ghost_skeleton.js",
    ghost_animation: "destiny_ghost_animation.js",
    ...
  }
}
```
Then if we ever get support for emotes (a guy can dream), they'd have their specific animation path in the translationBlock of the item definition.
